tests/Tests/500__jupyterhub/test-jupyterhub-api-telemetry.robot

```diff
@@ -0,0 +1,78 @@
+*** Settings ***
+Resource    ../../Resources/ODS.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/LaunchJupyterHub.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/LoginJupyterHub.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource    ../../Resources/Page/OCPDashboard/OCPMenu.robot
+Library     SeleniumLibrary
+
+*** Test Cases ***
+Verify Telemetry Data Is Accessible
+    [Tags]  Sanity
+    ...     ODS-347
+    Set Library Search Order  SeleniumLibrary
+    ${result} =   Is Usage Data Collection Enabled
+    Should Be True    ${result}
+    ${boolean} =    Checks SegmentKey Is Exposes In Jupyterhub API
+    Should Be True    ${boolean}
+    ${handle} =    Get Window Handles
```

Comment:
you could set this as suite variable so you don't need to pass it as argument.
As alternative, you could use the window labels instead of getting the handle (e.g., `Switch Window  title:Red Hat OpenShift Data Science Dashboard`

Reply:
Thanks 
I have chosen an alternative..